exhibitor_tls_artifacts/gen_artifacts.py

```diff
@@ -7,38 +7,42 @@
 
 
 @click.command()
-@click.argument('sans', nargs=-1)
+@click.option('--nodes', help='Comma separated list of master node ip addresses.')
 @click.option('-d', '--dir', help='Directory to put artifacts in.',
```

Comment:
Optional because this exists already, but it'd cool to see this have `type=click.Path(exists=True, dir_ok=True, file_ok=False)` (or similar)

Reply:
Good call. In this instance we want the artifact directory to `not` exist. We don't want to overwrite anything in `dir` and also we want to make sure that trusts are generated from scratch and not appended.

I will use the Path type, and will test that path does not exist at runtime. If `dir` does exist, I will print a message and exit the program with code 1.